### PR TITLE
New VASP Documentation

### PR DESCRIPTION
**Pull request by claralarson**

### CODE REVIEW

File: applications/vasp/README.md [hunk 1]
```diff
@@ -1,88 +1,71 @@
...
 
-VASP 5.4.4 is ~30% faster than VASP 6.1.1, but it doesn't support OpenMP. We run this benchmark using the same conditions as in Table 1 but using the following modules and report the results in Table 2.
-```
-module load vasp/5.4.4 intel-mpi/2020.1.217 mkl/2020.1.217
-```
+For information about accessing and running VASP on Vermilion, see the VASP section at the bottom of [this documentation](https://nrel.github.io/HPC/Documentation/Systems/Vermillion/running/). 
+
+## VASP Documentation
+
+This repo contains the results of two seperate VASP performance studies. The first, Performance Study 1, studies VASP performance on Eagle using the input files provided in the directory. The second, Performance Study 2, studies VASP performance on Eagle and Swift using benchmarks from the ESIF benchmarking suite, which can be found [here](https://github.com/NREL/ESIFHPC3/tree/master/VASP) or in the benchmarks folder in the Performance Harness 2 directory. Each study evaluates performance differently, as described below, and provides recommendations for running VASP most efficiently in the README files. The READMEs in each directory contain the following information.
```
Comment (yandthj) [inline applications/vasp/README.md:50]: seperate should be separate

Comment (claralarson) [reply applications/vasp/README.md]: done

File: applications/vasp/Performance Study 2/test [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (yandthj) [inline applications/vasp/Performance Study 2/test:1]: Blank file?

File: applications/vasp/Performance Study 2/benchmarks/README.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# VASP Benchmarking
+
+talk about goals of study, different input systems, systems run on, calculation performed (energy calculation), modules and version of VASP used
```
Comment (yandthj) [inline applications/vasp/Performance Study 2/benchmarks/README.md:3]: These look like notes, did you intend to put different text here?

File: applications/vasp/Performance Study 2/benchmarks/README.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+direct to VASP reframe repo in order to run your own tests using these (or other) input systems
+link to grey log to view others' VASP benchmarking results
+
+## Table of Contents
```
Comment (yandthj) [inline applications/vasp/Performance Study 2/benchmarks/README.md:7]: Are these links supposed to link to different pages or sections?

**Review by yandthj — CHANGES_REQUESTED** [review]

File: applications/vasp/Performance Study 2/README.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+
+On Eagle, the default build of VASP installed on the system is an Intel MPI version of VASP. The Intel MPI build was built with Intel compilers and the mkl math library, and it was accessed via the "vasp" module. It runs VASP 6.1.2. No Open MPI VASP build is accessible through the default modules on Eagle, but an Open MPI build can be accessed in an environment via "source /nopt/nrel/apps/210830a/myenv.2108301742, ml vasp/6.1.1-l2mkbb2". The OpenMPI build was compiled with gnu using gcc and fortran compilers and used OpenMPI's math libraries. It runs VASP 6.1.1.
+
+The [VASP repo](https://github.com/claralarson/HPC/tree/master/applications/vasp/VASP%20Recommendations) contains scripts that can be used to run the Intel MPI and Open MPI builds used in the study to perform calculations on Swift and Eagle. Additionally, the repo contains the VASP peroformance data collected, a notebook that analysis the performance data, data on CPU usage over time with different values of cpu-bind, and a notebook that analyses and visualizes the cpu-bind data.
```
[the remark this answers is not in the document]
Comment (claralarson) [reply applications/vasp/Performance Study 2/README.md:7]: done

File: applications/vasp/Performance Study 2/README.md [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+![Eagle GPU Bench 1 4x4x2](https://github.com/claralarson/HPC/blob/4837ef43a7d03b1b34a1e5ceda8dcbc64ca5b128/applications/vasp/VASP%20Recommendations/Images/Eagle_GPU_1_4x4x2.png)
+
+### MPI
+
+Intel MPI is recommended over Open MPI. Using an Intel MPI build of VASP and running over Intel MPI, Benchmark 2 ran in average of 50% of the time as the same calculations using an Open MPI build of VASP over Open MPI. For Benchmark 1, Intel MPI calculations ran in an average of 63.5% of the time as Open MPI calculcations. 
+
+Find scripts for running the Intel MPI and Open MPI builds of VASP in [this section](#Scripts-for-Running-VASP-on-Eagle).
+
+### --cpu-bind Flag
+The --cpu-bind flag changes how tasks are assigned to cores throughout the node. Setting --cpu-bind=cores or rank showed no improvement in the performance of VASP on 36 CPUs/node. Using 18 CPUs/node setting --cpu-bind=cores shows a small improvement in runtime (~5% decrease) using both Intel MPI and Open MPI. (See [cpu-bind analysis](https://github.com/claralarson/HPC/blob/a2a0b9eba1bf568b00e52cb06eac36253f8363c3/applications/vasp/VASP%20Recommendations/cpu-bind%20data/cpu-bind_VASP.ipynb) for info on the effect of cpu-bind)
```
[the remark this answers is not in the document]
Comment (claralarson) [reply applications/vasp/Performance Study 2/README.md:36]: Changed to "When running on 18 CPUs/node, setting --cpu-bind=cores shows a small improvement in runtime". Is this what you were referring to?

File: applications/vasp/Performance Study 2/benchmarks/README.md (hunk 3 above)
Comment (claralarson) [reply applications/vasp/Performance Study 2/benchmarks/README.md]: file shouldn't be there, deleted file

File: applications/vasp/Performance Study 2/benchmarks/README.md (hunk 4 above)
Comment (claralarson) [reply applications/vasp/Performance Study 2/benchmarks/README.md]: file shouldn't be there, deleted file

File: applications/vasp/Performance Study 2/README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (claralarson) [reply applications/vasp/Performance Study 2/README.md]: Need to update links after publishing (also check image links)

**Review by yandthj — APPROVED** [review]

**Review by arswalid — APPROVED** [review]
looks good

**Pull request merged**